spec/rubocop/config_loader_spec.rb

```diff
@@ -386,6 +387,14 @@
       )
     end
 
+    it 'loads configuration properly when it includes non-ascii characters ' do
+      create_file(configuration_path, ['# All these cops of mine are ❤',
```

Comment:
Can you duplicate the issue by appending `.force_encoding('binary')` on this one literal string?

Change the default encoding of all the spec files seems a bit heavy-handed.